frontend/karma.conf.js

```diff
@@ -8,28 +8,36 @@ module.exports = function (config) {
       // I18n.js is provided by the Asset pipeline,
       // which is unavailable for unit tests.
       // For testing, shim its functionality
-      'src/test/i18n-shim.ts',
       'node_modules/jquery/dist/jquery.js',
       // 'node_modules/angular-mocks/angular-mocks.js'
     ],
     frameworks: ['jasmine', '@angular-devkit/build-angular'],
     plugins: [
       require('karma-jasmine'),
       require('karma-chrome-launcher'),
+      require('karma-coverage'),
```

Comment:
From glancing at the source for `@angular-devkit/build-webpack`, `ng test` should append this plugin automatically if `"test"`/`"options"`/`"codeCoverage": true` is specified. However I couldn't get it to work.